README.md

```diff
@@ -1,12 +1,56 @@
 # bikes-for-refugees
-Exercise for HTML lesson 1
+Exercise for HTML & CSS - lesson 1
 
-Fork the repository to your personal Github account and use the following terminal commands to download the files to your projects folder.
+### Set up
 
-`cd ~`
+**Fork** this repository to your personal Github account (look for the 'fork' button on this page).
+
+Then, **clone** it to your machine, using the following terminal commands.
+
+>Go to the folder you wish to save the project in:
 `cd <your-project-directory>`
+
+>Clone the repo:
 `git clone git@github.com:<your_username>/bikes-for-refugees.git`
```

Comment:
Not all students have SSH keys configured and we had few problems in the last module because of that.
It might be better to clone with `https` here

Reply:
We also could link to https://codeyourfuture.github.io/syllabus-master/others/making-a-pull-request.html too?

Reply:
Good point! Thanks